src/main/java/frc/robot/commands/StorageID.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Storage;
+
+public class StorageID extends CommandBase {
+
+    private Storage storage;
+
+    public StorageID(Storage storage) {
+        this.storage = storage;
+        addRequirements(storage);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
```

Comment:
I think this needs more work, consider that this method will likely be called every 20ms during the match. 

Since this clears all remembered state as soon as it starts, this will cause the conveyors to behave erratically, because they base their movement solely on the current value of the sensors. 

Instead, you need to set "goals" for it achieve, effectively. As in, "once I see this, move till this happens," and not to forget the previous state, since that's invaluable in determining what's going on in the robot, since we don't have continuous tracking of the balls.